doc/ref/chap-expr-lang.md

```diff
@@ -344,6 +344,8 @@ and `List` objects:
     ysh$ = M
     (List)   ["one", "two", "three", "4"]
 
+String concatenation is slow, list appending is faster for looping.
```

Comment:
I think it would be clearer to say something like:

"Note: Avoid repeated string concatenation in a loop.  It's algorithmically faster to append to a list, and then join() at the end."

Actually I think we couild put that in `doc/ysh-faq`, and then **link** to it from the reference.